electron/app/menu.ts

```diff
@@ -231,6 +234,7 @@ const editMenu = (state: RootState, dispatch: MainDispatch): MenuItemConstructor
         accelerator: hotkeys.APPLY_SELECTION.key,
         enabled: Boolean(state.main.selectedResourceId),
```

Comment:
```suggestion
        enabled: Boolean(state.main.selectedResourceId) && Boolean(isKubeConfigPathValid),
```

We could just use the enabled property for making the option disabled if the kubeConfig path is not valid